lib/OpenQA/Task/Git/Clone.pm

```diff
@@ -65,7 +65,9 @@ sub _git_clone ($app, $job, $ctx, $path, $url) {
 
         my $origin_url = $git->get_origin_url;
         if ($url ne $origin_url) {
-            $ctx->warn("Local checkout at $path has origin $origin_url but requesting to clone from $url");
+            $ctx->info(
+"Local checkout at $path has origin $origin_url but requesting to clone from $url. This is not that bad."
```

Comment:
Suggestion by @okurz ;-)
```suggestion
"Local checkout at $path has origin $origin_url but requesting to clone from $url. This is fine.🔥"

```

Reply:
Sorry, that was just meant to be a joke, not a real suggestion

Reply:
Pssst. Don't prevent the opinion to see what we can break with UTF-8 characters on the logfile ;)